### PR TITLE
Batch writes tutorial

### PR DESCRIPTION
**Pull request by neelp-git**

### CODE REVIEW

File: notebooks/java/batch_ops.ipynb
Comment (reugn) [inline notebooks/java/batch_ops.ipynb:804]: Missing backtick

File: notebooks/java/batch_ops.ipynb
Comment (reugn) [inline notebooks/java/batch_ops.ipynb:86]: replace `/home/jovyan` with `~`

**Review by reugn — APPROVED** [review]
Well done.

**Comment by neelp-git** [comment]
Thanks @reugn . I have to add a link to the blog post which should not require another review. Will merge after that.

**Pull request closed without merging**
